ocaml/idl/datamodel_errors.ml

```diff
@@ -1905,7 +1905,13 @@ let _ =
   error Api_errors.bundle_repo_not_enabled []
     ~doc:"Cannot sync bundle as the bundle repository is not enabled." () ;
   error Api_errors.can_not_sync_updates []
-    ~doc:"Cannot sync updates as the bundle repository is enabled." () ;
+    ~doc:"The current enabled repositories are not supported to sync updates."
```

Comment:
Do you mean: the currently enabled repositories do not support synchronization of updates?

Reply:
Yes. I updated it.